runners/direct-java/src/main/java/org/apache/beam/runners/direct/portable/QuiescenceDriver.java

```diff
@@ -154,19 +155,17 @@ private void applyUpdate(
   /** Fires any available timers. */
   private void fireTimers() {
     try {
-      for (FiredTimers<AppliedPTransform<?, ?, ?>> transformTimers :
-          evaluationContext.extractFiredTimers()) {
+      for (FiredTimers<PTransformNode> transformTimers : evaluationContext.extractFiredTimers()) {
+        checkState(false, "fireTimers not yet supported");
```

Comment:
throw new UnsupportedOperationException?

Reply:
Done. This will actually break earlier, but there are no users so that's fine until it's fully implemented.